ortools_result.proto

```diff
@@ -11,20 +11,24 @@ message Activity {
   int32 alternative         = 5;
   int32 current_distance    = 6;
   string id                 = 7;
+  int64 lateness            = 8;
+  repeated float overloads  = 9;
 }
 
 message Costs {
-  float fixed = 1;
-  float distance = 2;
-  float distance_balance = 3;
-  float distance_fake = 4;
-  float distance_order = 5;
-  float time = 6;
-  float time_balance = 7;
-  float time_fake = 8;
-  float time_order = 9;
-  float time_without_wait = 10;
-  float value = 11;
+  int64 fixed = 1;
+  int64 distance = 2;
+  int64 distance_balance = 3;
+  int64 distance_fake = 4;
+  int64 distance_order = 5;
+  int64 time = 6;
+  int64 time_balance = 7;
+  int64 time_fake = 8;
+  int64 time_order = 9;
+  int64 time_without_wait = 10;
+  int64 value = 11;
+  int64 lateness = 12;
+  int64 overload = 13;
```

Comment:
Comment as a reminder : Costs should be float because we divide by `CUSTOM_BIGNUM`. @braktar  I believe some part of this division is automatic (in the sense that it is inside sub-functions), you might want to check how it is implemented at the moment before dividing every cost by `CUSTOM_BIGNUM`...